src/lando/api/legacy/uplift.py

```diff
@@ -59,17 +64,24 @@ def get_uplift_request_form(revision: dict) -> Optional[str]:
     return bug
 
 
-# @cache.cached(
-#    key_prefix="uplift-repositories"
-# )
-def get_uplift_repositories(phab: PhabricatorClient) -> list:
-    repos = phab.call_conduit(
-        "diffusion.repository.search",
-        constraints={"projects": ["uplift"]},
-    )
-
-    repos = phab.expect(repos, "data")
-
+def get_uplift_repositories() -> list:
```

Comment:
Is there a type that we could make it a `list[]` of?

Reply:
I can add that.

Side note: this is an early version of uplift functionality, and as part of updating the repo with all the changes in the lando-api and lando-ui repos, this code will be affected.